config/locales/ko/course/assessment/answer/programming_auto_grading.yml

```diff
@@ -0,0 +1,31 @@
+ko:
+  course:
+    assessment:
+      answer:
+        programming_auto_grading:
+          grade:
+            evaluation_failed_syntax: >
+              코드가 올바르게 평가되지 않았습니다. 구문 오류가 있거나 지원되지 않는 라이브러리가
+              사용되었을 수 있습니다. 코드를 다시 확인해 주세요.
+            evaluation_failed_time_or_memory: >
+              코드가 할당된 시간 또는 메모리 제한 내에 완료되지 않았습니다.
+            time_limit_error: >
+              시간 제한 초과 (TLE) 오류
+            memory_limit_error: >
+              메모리 제한 초과 (MLE) 오류
+          job:
+            failure:
+              time_limit_breached: >
+                평가기가 실행 중에 종료되었습니다. 이 문제의 시간 제한이 coursemology에서
+                허용되는 평가기 시간 제한을 초과했을 수 있습니다. 이 문제에 대해 강사에게 문의하시고, 문제가 계속되면 coursemology
+                관리자에게 문의하세요.
+              timeout_error: >
+                작업 실행 시간이 길어져 타임아웃 오류가 발생했습니다. 몇 분 후에 다시 시도해 주세요. 문제가
+                계속되면 강사나 coursemology 관리자에게 문의하세요.
+              container_unreachable: >
+                평가기를 위한 도커 생성 중 오류가 발생했습니다. 서버 작업 부하가 높기 때문일
+                수 있습니다. 몇 분 후에 다시 시도해 주세요. 문제가 계속되면 강사나 coursemology 관리자에게 문의하세요.
+              generic_error: >
+                오류가 발생했습니다. 서버가 바쁠 수 있으니 몇 분 후에 다시 시도해 보세요. 동일한 문제가
+                계속되면, coursemology 관리자에게 문의하고 이 페이지의 URL을 제공하며 다음 오류를 함께 제공해주세요: '"%{error}"
+                
```

Comment:
remove trailing spaces? Just leave a single trailing newline.

Reply:
Thats the syntax of "folded block scalar." of yaml: https://yaml-multiline.info/